packages/fluentui/react-northstar/src/index.ts

```diff
@@ -315,6 +315,8 @@ export * from './components/Card/CardColumn';
 export { default as CardColumn } from './components/Card/CardColumn';
 export * from './components/Card/CardExpandableBox';
 export { default as CardExpandableBox } from './components/Card/CardExpandableBox';
+export { default as Datepicker } from './components/Datepicker/Datepicker';
```

Comment:
Let's export this component as `unstable__Datepicker` for now, so potential users know that it's not production ready yet :)

Reply:
Maybe `Unstable_DatePicker`, capitalized so that it indicates that it is a component name.